content/metadata.yaml

```diff
@@ -110,6 +110,12 @@ author_info:
 #   twitter: null
 #   affiliations:
 #   - Case Western Reserve University
+- name: J. Brian Byrd
+orcid: 0000-0002-0509-3520
```

Comment:
Can you add two spaces before `orcid:` and each of the following lines?

You can do this by clicking the pencil icon